### DISCUSSION TITLE
Problemas con mvn test

### DISCUSSION THREAD

**Discussion by JaviGatica** [dds-jv/foro#356] [Consultas de herramientas] [answered]
Buenas estoy teniendo problemas al querer correr mvn test:

![image](https://user-images.githubusercontent.com/82297774/165597717-79305172-f797-4d8e-b918-dd05803e0983.png)

**Comment by gastonprieto** [comment, marked as answer]
Hola @JaviGatica 

Fijate en https://github.com/dds-jv/foro/discussions/357 que respondi algo similar

**Reply by JaviGatica** [reply]
Hola de nuevo

![image](https://user-images.githubusercontent.com/82297774/165654567-9cdc7d4b-a682-4982-b6ec-dd0fbc422688.png)

Asumo que el problema en #357 era de las versiones de java y maven
En mi caso creo tenerlo bien instalado

**Reply by gastonprieto** [reply]
claro, pero fijate que dice en Java version, leete toda esa linea, entre ese mensaje y el otro issue esta lo que comentaba
